app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/templates.tsx

```diff
@@ -1,62 +1,137 @@
 /** @file Renders the list of templates that can be used to create a project from. */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import * as react from "react";
+import * as React from 'react'
 
-const templates = [
-  { title: "New empty project", templateName: undefined },
-  { title: "Colordo COVID", templateName: "Colorado_COVID" },
-  { title: "KMeans", templateName: "Kmeans" },
-  { title: "NASDAQ Returns", templateName: "NASDAQ_Returns" },
-  { title: "Restaurants", templateName: "Orders" },
-  { title: "Github Stars", templateName: "Stargazers" },
-];
+// =================
+// === Templates ===
+// =================
 
-interface Props {
-  onChange: (templateName: string | undefined) => void;
+/** Define the template type to render */
+interface Template {
+    title: string
+    description?: string
+    templateName?: string
 }
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-const Templates = (props: Props) => {
-  const { onChange } = props;
+/** Specifically refers to the creation of empty projects */
+const emptyTemplate = { title: 'New empty project' }
```

Comment:
Because this is a special create an empty project, the others are templates.
If there are any changes afterward it is easy to modify or determine if it is the special one.

Reply:
fair enough